quantities/dimensionality.py

```diff
@@ -9,6 +9,8 @@
 from .registry import unit_registry
 from .decorators import memoize
 
+_np_version = tuple(map(int, np.__version__.split('.')))
```

Comment:
This is now breaking the tests in MNE:

```
     _np_version = tuple(map(int, np.__version__.split('.')))
E   ValueError: invalid literal for int() with base 10: 'dev0+git20240824'
```

x-ref: https://github.com/mne-tools/mne-python/pull/12815

Reply:
Are you all testing against a development version of NumPy? I don't think @bjodah or I thought about that in these PRs for parsing the numpy version. I guess a version parse instead would be better if you want to defend against dev version testing.

Reply:
MNE is testing against stable and the development version of `numpy`, all my projects are also tested against both, and I'm probably not the only one ;) 
Quick fix in https://github.com/python-quantities/python-quantities/pull/238 but maybe you want something a bit more elaborate :smiley: